rmb-sdk-go/peer/connection.go

```diff
@@ -73,7 +81,28 @@ func (c *InnerConnection) reader(ctx context.Context, cancel context.CancelFunc,
 	}
 }
 
-func (c *InnerConnection) loop(ctx context.Context, con *websocket.Conn, output, input chan []byte) error {
+func (c *InnerConnection) send(ctx context.Context, data []byte) error {
+	if c.busy {
```

Comment:
I suggest u use an atomic bool like here https://pkg.go.dev/sync/atomic#Bool so we don't run into race conditions